modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/build.gradle

```diff
@@ -45,10 +53,12 @@ wrapSoyAlloyTemplate {
 	moduleName = "${artifactId}-form-field-template"
 	namespace = "ddm"
 }
-#if (${projectType} != "workspace")
+#end
 
+#if (${projectType} != "workspace")
 repositories {
 	maven {
 		url "https://repository-cdn.liferay.com/nexus/content/groups/public"
 	}
-}#end
\ No newline at end of file
+}
```

Comment:
We need to remove this extra newline.  Try to match the formatting that you see in other templates.